content/2018/03/rising-e-commerce-fraud-trends-and-how-to-combat-them.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Rising E-Commerce Fraud Trends and How to Combat Them"
 date: "2018-03-22"
 categories:
-  - "online-fraud-prevention"
+  - "Online Fraud Detection"
```

Comment:
@christophermluna Can you please confirm whether this updated category name is correct?

Reply:
this looks right as well